### PR TITLE
Bug 1138147 - Prevent fuzzing from triggering events in the current app ...

### PR DESCRIPTION
**Pull request by etiennesegonzac**
...when taping the SHB background. r=gmarty

### CODE REVIEW

**Comment by try-server-hook** [comment]
[etiennesegonzac Etienne Segonzac (etiennesegonzac)](https://github.com/etiennesegonzac) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2097b356b264)

File: apps/system/js/software_button_manager.js [hunk 1]
```diff
@@ -238,7 +238,8 @@
         this.screenElement.classList.add('software-button-enabled');
         this.screenElement.classList.remove('software-button-disabled');
 
-        this.homeButtons.forEach(function sbm_addTouchListeners(b) {
+        this.element.addEventListener('click', this._emptyEventHandler);
+        this.homeButtons.forEach(function(b) {
```
Comment (gmarty) [inline apps/system/js/software_button_manager.js]: nit: why get rid of the named function?

File: apps/system/js/software_button_manager.js [hunk 2]
```diff
@@ -248,7 +249,8 @@
         this.screenElement.classList.remove('software-button-enabled');
         this.screenElement.classList.add('software-button-disabled');
 
-        this.homeButtons.forEach(function sbm_removeTouchListeners(b) {
+        this.element.removeEventListener('click', this._emptyEventHandler);
+        this.homeButtons.forEach(function(b) {
```
Comment (gmarty) [inline apps/system/js/software_button_manager.js]: same

Comment (etiennesegonzac) [inline apps/system/js/software_button_manager.js]: it's not needed anymore, but you're right: let's keep the patch small :)

**Comment by try-server-hook** [comment]
[etiennesegonzac Etienne Segonzac (etiennesegonzac)](https://github.com/etiennesegonzac) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=74273590a164)

**Pull request merged**